code/modules/martial_arts/hokuto_shinken.dm

```diff
@@ -0,0 +1,115 @@
+#define SOKIN_JIZAI_KYAKU_COMBO "HHD"
+#define GOSHI_RETSU_COMBO "DDHD"
+#define GEKITSUI_SHI_COMBO "HDHDH"
+
+/datum/martial_art/hokuto_shinken
+	name = "Hokuto Shinken"
+	deflection_chance = 100
+	help_verb = /datum/martial_art/hokuto_shinken/proc/hokuto_shinken_help
+
+
+/datum/martial_art/hokuto_shinken/proc/check_streak(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(findtext(streak,SOKIN_JIZAI_KYAKU_COMBO))
+		streak = ""
```

Comment:
Why are you emptying streak here?

Reply:
That is just how the martial art moves are designed, the streak is always emptied when one procs. I assume that is to stop spammable moves, or to keep the streak clear after doing a move.